docker-images/python-36/Dockerfile

```diff
@@ -13,5 +13,5 @@ RUN curl -sLo /tmp/sonar-scanner-cli.zip https://dl.bintray.com/sonarsource/Sona
 COPY requirements.txt ${APP_ROOT}/src
 RUN source /opt/app-root/etc/scl_enable && \
     set -x && \
-       pip install -U pip==18.1 setuptools wheel && \
+    pip install -U pip setuptools wheel && \
```

Comment:
I had to freeze pip version a while back because they introduced some bug.  Seems to be happy and fixed now